<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>566</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>566</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>566</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>SpeakyNotes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 67, 118);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(6, 167, 125);
}
QPushButton:hover{
	background-color: rgb(9, 255, 190);
}
</string>
    </property>
    <property name="text">
     <string>Adicionar Lembrete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:10px;
background-color: rgb(215, 106, 3);}</string>
    </property>
    <property name="text">
     <string>Excluir Lembrete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:10px;
background-color: rgb(206, 125, 165);}</string>
    </property>
    <property name="text">
     <string>Listar Lembretes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comandButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(108, 255, 17);
}</string>
    </property>
    <property name="text">
     <string>Comandos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(245, 213, 71);
}</string>
    </property>
    <property name="text">
     <string>Ajuda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(136, 0, 68);
}</string>
    </property>
    <property name="text">
     <string>Sobre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="micButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(230, 230, 230);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
